en/sensor/rangefinders.md

```diff
@@ -78,16 +78,27 @@ The uLanding Radar is not present in "most" firmware by default and must be star
 The *LeddarOne* must typically be added to firmware before it can be used. For setup/usage information see: [LeddarOne](../sensor/leddar_one.md).
 
 
-### TFmini 
+### TFmini
 
-The [Benewake TFmini LiDAR](http://www.benewake.com/en/tfmini.html) is a tiny, low cost, and low power LIDAR with 12m range. 
-
-The sensor should be connected to the SERIAL 4/5 port on Pixhawk devices. It is enabled by setting parameter [SENS_EN_TFMINI](../advanced_config/parameter_reference.md#SENS_EN_TFMINI) > 0.
+The [Benewake TFmini LiDAR](http://www.benewake.com/en/tfmini.html) is a tiny, low cost, and low power LIDAR with 12m range.
 
 > **Note** This lidar is currently in master, and should appear in releases from PX4 v1.8.
 
+In general the TFmini can be used by setting the parameter [SENS_EN_TFMINI](../advanced_config/parameter_reference.md#SENS_EN_TFMINI) > 0.
+
+For boards based on [PX4FMU_V3](../flight_controller/pixhawk_series.md#fmu-versions) firmware (only) the sensor can be connected to the SERIAL 4/5 port.
+
+For the [Intel Aero](../flight_controller/intel_aero.md), the sensor can be connected to the TELEMETRY port.
+
+For other Pixhawk based boards the parameter [SYS_COMPANION](../advanced_config/parameter_reference.md#SYS_COMPANION) has to be 0 and the sensor can be connected to TELEM2.
+
+To use the TFmini on other boards/ports, the driver can be started in a [shell](https://dev.px4.io/en/debug/system_console.html#mavlink-shell) with

```

Comment:
I like this as an alternative, but it would be even more of a pain than using an SD card if you had to do this on number of devices. So suggest we have both approaches listed.